documentation/source/install.rst

```diff
@@ -0,0 +1,124 @@
+.. _install:
```

Comment:
Nit: would marginally prefer `installation.rst`. I'll see what other projects do for this first.

Reply:
Google says ``"source/installation.rst"`` wins with 2.1k vs. 1.6k hits.
But we also have ``install_devel.rst``, should this then be called ``installation_devel.rst`` or ``installation_development.rst``?

(While we are at it, Sphinx' default for the directory would be ``docs/source/``.)

Reply:
Not a big deal - although it is tempting to rename the folder to "docs". Thanks for collecting the data!